### ISSUE TITLE
`Couldn't Find Transcript for NPTEL2020 - Indian English Speech Dataset

### ISSUE THREAD

**Issue by riffs14** [AI4Bharat/NPTEL2020-Indian-English-Speech-Dataset#17]
Hi, I have downloaded the dataset, But I couldn't find the transcript for any of the audio file. However the sample dataset was well documented and has transcript. Please suggest ways to get the transcript for the audio files in the dataset.

Thanks :-)

**Comment by Prem-kumar27** [comment]
Hi @riffs14 

Thanks for your interest in this dataset. If I remember correctly, the dataset folder would constitute 3 primary folders - wav, txt and metadata. wav folder will contain the audio clips and the txt folder would contain all the transcripts.

I think your downloaded data might be partially downloaded. Please check if every zip file is correctly downloaded and then extract them. Please check and let us know if you face issues then.

**Comment by riffs14** [comment]
Hi @Prem-kumar27 
Thanks . I'll let you know if the problem persist.

**Comment by kafan1986** [comment]
@riffs14 Where you able to find the transcript data inside the dataset?

**Comment by riffs14** [comment]
@kafan1986 no, there is no transcript file in train, validation or test data. Kindly check from your end.

**Comment by kafan1986** [comment]
@riffs14 There is no transcript files inside the zip directories. Maybe it is present in the zip files in torrent version of the download. Can you share them separately? for Valid, train and test? As redownloading 100+ GB is too much for me.

**Comment by Prem-kumar27** [comment]
@kafan1986 @riffs14 I downloaded the test set and extracted it to check if there is any issue in the dataset. 
The dataset seems to be fine.
>metadata , txt , wav

It had all three folders with all the files. So, I strongly think your files must be partially downloaded. 
Instead of running the script, can you try running the download script commands line by line? 

Otherwise please use the torrent version as it is of small size comparatively. Please let us know if you face any issues.
